Automattic/woocommerce-payments
# Protect account status usage during Overview page init

Fixes #4204 

#### Changes proposed in this Pull Request

Account status was being used without caution during Payments > Overview page init.

In case the account data is not defined, there is a breaking JS error when accessing the Overview page.
We protect the usage of the account status to avoid this breaking error and show the page as expected.

Before:
![image](https://user-images.githubusercontent.com/28830738/165716897-9a32cba8-f53a-4fa7-8db1-f950af570d3e.png)

After:
![image](https://user-images.githubusercontent.com/28830738/165717509-198fd899-5d22-40d0-a204-f022e1009250.png)

#### Testing instructions

- First create a WCPay account using your local server
- Stop using your local server (untick "Enable API redirection" in WCPay Dev)
- The account cache content should shows data = NULL in WCPay Dev
- Visit Payments > Overview
- You can see the page and there is an error message in "Account details" section regarding the account connection

-------------------


**Post merge**
